pyvista/core/utilities/helpers.py

```diff
@@ -110,6 +110,9 @@ def wrap(
         # Return object if it is already wrapped
         return dataset  # type: ignore
 
+    if isinstance(dataset, list):
+        dataset = np.array(dataset)
```

Comment:
We could have a list of `DataSets` that we should maybe coerce into a `MultiBlock`.  Also if the `isinstance` check is changed to `Sequence` a `MultiBlock` could be here.  We need to handle all these edge cases here.

Reply:
Good point. I am considering whether we can use `validate_array`.
https://github.com/pyvista/pyvista/blob/1d27d8f331c75929a89e444770c21d81f031a25b/pyvista/core/input_validation/validate.py#L37

Reply:
It seems to work fine! But maybe we need to wait for #5226.

Reply:
The scenario in which this creates a problem is when the `pyvista._wrappers` has been overwritten for `vtkMultiBlockDataSet`.  Then when any piece of internal or external code calls `pv.wrap(multi_block_mesh)`, where `multi_block_mesh` is type `pv.MultiBlock`, it will fail on this line.  We need to handle this edge case here in my opinion, but we don't necessarily need to add in support for something like `[mesh_1, mesh_2, mesh_3]` to be wrapped to a `pv.MultiBlock` at this time.

Reply:
The reason is because `pv.MultiBlock` inherits from `collections.abc.Sequence`.